crates/huub/src/propagator/array_var_int_element.rs

```diff
@@ -19,20 +19,22 @@ use crate::{
 pub(crate) struct ArrayVarIntElementBounds {
 	vars: Vec<IntView>,                // Variables to be selected
 	y: IntView,                        // The selected variable
-	idx: IntView,                      // Variable that stores the index of the selected variable
+	index: IntView,                    // Variable that stores the index of the selected variable
 	action_list: Vec<(u32, IntEvent)>, // List of x variables that have been modified since the last propagation
```

Comment:
Action list does not seem to be actually used. Probably just remove it and the usage of it

Reply:
updated.